src/components/controls/styles.js

```diff
@@ -30,6 +30,23 @@ export const HeaderContainer = styled.div`
   margin-bottom: 5px;
 `;
 
+export const PanelSectionContainer = styled.div`
+  // Less padding is necessary on the top because there is already some space
+  // from HeaderContainer's top margin.
+  padding-top: 2px;
+  padding-bottom: 8px;
+
+  // Add borders to delineate panel sections from other sections.
+  border-top: 1px solid ${(props) => props.theme.color};
```

Comment:
This is personal preference, but I'd prefer a more subtle colour here (especially when using the nextstrain.org theme). Something like `props.theme.unselectedBackground` works, but it's not exactly great naming. What do you think? It's not major, and it's not something that should hold up this PR.

Reply:
Good suggestion. I don't think the border color should be coupled with either the font color (which I currently have) or the unselected background color. Some options:

1. Add an additional entry `borderColor`. This would be the easiest change, but thinking long-term it's one more specific thing that needs to be documented and defined.
2. Rename `unselectedBackground` to encompass both the border and toggle background. I can't think of a good name.
3. Use a scale `neutral1`, `neutral2`, … to go from lightest neutral to darkest neutral (or inverse for dark mode). I think the hard part would be coming up with the number of shades in the scale ([this article says 5 shades is good](https://uxdesign.cc/defining-colors-in-your-design-system-828148e6210a)), but should be more future-proof and can be extended to other elements where neutrals are currently hardcoded ([1](https://github.com/nextstrain/auspice/blob/ba7e1c20d463f69d570cd46cc00e9bd9e51d49fa/src/components/controls/styles.js#L41), [2](https://github.com/nextstrain/auspice/blob/ba7e1c20d463f69d570cd46cc00e9bd9e51d49fa/src/components/controls/slider.js#L45), [3](https://github.com/nextstrain/auspice/blob/ba7e1c20d463f69d570cd46cc00e9bd9e51d49fa/src/components/framework/fine-print.js#L29)).

Reply:
Long term 3 is the best option if we really want to help whitelabelling efforts, but it's a large amount of work and I think needs a handful of customised versions to develop across. 

For this task I'd suggest renaming `unselectedBackground` to `alternateBackground`, and allowing background compatibilty by renaming the former to the latter on load (e.g. in `styles.js:49`)

Reply:
Sounds good. Done in 67bffee2b0dd80066b32eba028d5b0b6b701d7ec...aa6e3df788853bb12cde4c8968847c5cc05dfbc3